src/SourceBrowser.Site/Scripts/upload.js

```diff
@@ -0,0 +1,29 @@
+﻿$(document).ready(function () {
+    resetUI();
+    registerButtonClickEvent();
+});
+
+function registerButtonClickEvent() {
+    getButtonFromDom().click(prepareForChange);
+}
+
+function prepareForChange() {
+    getButtonFromDom().prop("disabled", true);
+    getMessageDivFromDom().css("visibility", "visible");
+}
+
+function resetUI() {
+    getButtonFromDom().prop("disabled", false);
+    getMessageDivFromDom().css("visibility", "hidden");
```

Comment:
`Content\Site.css` is a generic place where we dumped CSS.
Then we started to specialize into `Content\sourcebrowser-browse.css` and `Content\sourcebrowser-codebrowser.css`. I think either `Site.css` or new `sourcebrowser-upload.css` would be good places for CSS.